package.json

```diff
@@ -31,7 +31,8 @@
     "package-win": "npm run build && electron-builder --win --x64",
     "package-mac": "npm run build && electron-builder --mac",
     "package-linux": "npm run build && electron-builder --linux",
-    "package-all": "npm run build && electron-builder -mwl"
+    "package-all": "npm run build && electron-builder -mwl",
+    "spectron-test": "yarn build && mocha -r esm"
```

Comment:
We've migrated from mocha to jest a while ago. I'm just wondering whether the spectron tests will work with jest, or is it only possible with mocha?

Reply:
whoops, didn't realize that, totally will work with jest, will make it so!